pkg/source/source.go

```diff
@@ -20,6 +22,7 @@ type source interface {
 	Title() string
 	Output() string
 	Ignore() bool
+	OmitNotice() bool
```

Comment:
I don't think `OmitNotice()` needs to be part of this interface.

Reply:
- [ ] remove `OmitNotice()`